pilot/docker/envoy_pilot.yaml.tmpl

```diff
@@ -45,6 +45,52 @@ static_resources:
     http2_protocol_options: {}
     name: mixer_report_server
 {{- if .ControlPlaneAuth }}
+{{- if .SDSEnabled }}
+    tls_context:
```

Comment:
May be make this a named template so you can reuse.

Basically, define a named template like below (the whole thing can be under a separate file, best would be `istio/templates/_sdsConfig.tpl`)

```
{{- define "sdsConfig" }}
.... // your code
{{- end }}
```

Then here you can simply include them

{{ include "sdsConfig" . }}

(you can do better with passing only the variables needed)

Reply:
I tried (putting something below in bootstrap yaml as other .tpl files),   
```
{{- include "sdsconfig.yaml.tpl" }
``` 
then hit below error, I think current include tpl files works for helm, but bootstrap files is different; so prefer to put as it in this PR as existing tlscontext does. 
```
2019-04-29T21:05:28.269349Z	error	template: envoy_pilot.yaml.tmpl:49: function "include" not defined
Error: template: envoy_pilot.yaml.tmpl:49: function "include" not defined
```

Reply:
You should include the 'defined function', not the file. I.e, you need to wrap your code/template between a `{{- define "your-template-name" }}`, and then include them with:

```
{{ include "your-template-name" . }}
```

Also note the indentation should be correct. The easiest way is writing your template as if they are started from 0, and add indent in the include clause

```
{{ include "your-template-name" . | nindent 8 }}
```

Reply:
Take back my comment. I thought this is helm template.

Reply:
New installer doesn't share templates - each component is independent.